core/domain/collection_jobs_one_off.py

```diff
@@ -79,9 +80,16 @@ def map(item):
                 'Update collection schema version to %d.' % (
                     feconf.CURRENT_COLLECTION_SCHEMA_VERSION))
             yield (
-                CollectionMigrationJob._MIGRATED_KEY,
+                CollectionMigrationOneOffJob._MIGRATED_KEY,
```

Comment:
I suggest handling this in the same way as the deleted case -- use counts.

(Ditto in other jobs, if applicable.)

Reply:
This was done too, sorry I forgot to mention before.